app/src/js/reducers/reconciliation-reports.js

```diff
@@ -78,4 +86,7 @@ export default createReducer(initialState, {
   [CLEAR_RECONCILIATIONS_FILTER]: (state, action) => {
     state.list.params[action.paramKey] = null;
   },
+  [RECONCILIATION_DELETE]: createSuccessReducer('deleted'),
```

Comment:
Will these reducers change anything?

Reply:
I don't think so. I had added them earlier and forgot to remove them. They match the other delete reducers but they probably aren't necessary. I can get rid of them

Reply:
Done https://github.com/nasa/cumulus-dashboard/pull/780/commits/b0656ad4e29d4e09e4dc3059fc536111c137d0f5